docs/3-integration-with-ecom/commercetools/Commercetools-api/add-line-item-to-cart.api.mdx

```diff
@@ -0,0 +1,217 @@
+---
+id: add-line-item-to-cart
+title: "Add Line Item to cart"
+description: "Add Line Item to cart"
+sidebar_label: "Add Line Item to cart"
+hide_title: true
+hide_table_of_contents: true
+api: eJyNVE1v2zAM/SuGThvgfO3oW9fuEKDAiiw7ZTkoFhOrtSVVkrMahv/7SMlJ7KRblwCJJZF85HvPapmF1xqc/6pFw7I2LKUFwTJva0hZrpUH5emIG1PKnHup1ezZaUV7Li+g4vTkGwMsY3r3DLlnKTNWG7BegqPTI1gnY0of6LyV6oCB8MYrU9JW2+bc+kkf23WsSxnPCc8NErm1vME86aFyHyPHAv8G5kI8SgVLrEiYWEDUuV+Kj9rtAydSxGaP3Equ/iPxHBjzXmtcSN8M0iTSfgA7zFt09EkZRoYyd5euV1FEFgMsOIOcxfm/zOf0J8DlVprIBaO8hAhMcHQQiavzHJzb12XZMCoxDkekJKQQVuJ1QjoR09zyCjzqxbJNiz1jrOG+wCOFB7gKgkqBGzdO6YkJ/Y49NwZfF4j7kOh94vGJCk5Zt00ZAhdaEKJ2sRkEztiMItys7ZE7ggZ7PPVY2zIoUGBS182ChuSbyQs0JAYWJm+vLm/FtxP/AxffWnXg1M3FdCNnjYx17Z6Rea4cMjTIotsiY1LtdaCyt8K9riqwOay1Ll1y97SkeudmF9P5dB6MjUNXPLwMvT5/U3YkwVkyD29+ZkqOQmO1QGXb874JUjtMzU6aI5XEMp217Y47+GlLnGZL44DFWTbbfuodDYG0CenoGRnY89JdO6G9XEbs06r3zOfkPX+k73d8ukFUE/gua1rhI2o/8GpguAAu8PWjruLpfcSerKnGJfvmUhxZ8+n7jzUG7/rLtdKCciz/jZv0m7Ff+MWFNudrLuy3rOTqUPMDxce69PkDVq305w==
```

Comment:
i don't like to store token here. we could move it to variables. also a question about could it be expired?

Reply:
see that it different for other files